remarkapy/api.py

```diff
@@ -182,12 +196,36 @@ def _register_device(self, code: str):
             "secret": ""
         }
 
+        headers = {'User-Agent': 'desktop/3.14.0.887 (macos 15.0)'}
+
         url = URLS.REGISTER_DEVICE
-        response = self._post(url, headers=headers, json=data)
+        response = self._post(url, headers=headers, data=data, timeout=20)
```

Comment:
Why specify the timeout explicitly? Does the server do something weird here?

Reply:
correct, that request would fail with the default timeout. It looks it's the expected behavior as even the RM Desktop apps take up to 10 seconds to pair

Reply:
wild! this is a weird software ecosystem :)